releasenotes/notes/flask-http-helpers-0167071f46a36e00.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations:
+  - |
+    flask: Use :ref:`HTTP Custom Error Codes <http-custom-error>` instead of ``ddtrace.config.flask['extra_error_codes']``.
```

Comment:
Does this link actually work when we build the docs? (Not sure if the RST gets rendered or not)

Reply:
Yep, the link works!

https://449008-61572326-gh.circle-artifacts.com/0/tmp/docs/release_notes.html#deprecation-notes